.github/workflows/benchmark_wasm_client.yml

```diff
@@ -97,10 +97,11 @@ jobs:
             echo "COMMIT_HASH=$(git describe --tags --dirty)";
           } >> "${GITHUB_ENV}"
 
-      - name: Set up home
-        # "Install rust" step require root user to have a HOME directory which is not set.
-        run: |
-          echo "HOME=/home/ubuntu" >> "${GITHUB_ENV}"
+      # TODO tester de virer cette merde (dixit Arthur le chef, c'est pas moi qui l'ai dit)
```

Comment:
hmm ?

Reply:
Leftovers of this debugging nightmare, my bad.